apps/protocol-frontend/src/contexts/UserContext.tsx

```diff
@@ -313,21 +294,21 @@ export const UserContextProvider: React.FC<UserContextProps> = ({
         createWaitlistUser,
         disconnectLinear,
         getAllDaos,
-        isUserActivityTypesLoading,
+        govrnProtocol,
         isUserLoading,
         getUserDaos,
         setAllDaos,
-        setUserActivityTypes,
+        setGovrnProtocol,
```

Comment:
I don't think we need this in the values, the context should probably only handle the setting operation